### PR TITLE
Worked on the tooltip issue on dark mode

### PR DESCRIPTION
**Pull request by Sanjeev1308**

### CODE REVIEW

File: parts/Compound/CompoundDialogs/CompoundSlider/index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const useStyles = makeStyles((theme) => ({
   },
   button: {
     color: theme.palette.text.primary,
-    backgroundColor: theme.custom.palette.blueButton,
+    backgroundColor: theme.custom.paletteblueContainer,
```
Comment (prodvdc) [inline parts/Compound/CompoundDialogs/CompoundSlider/index.js]: Is this correct? or dot is missing?

**Review by prodvdc — APPROVED** [review]
LGTM!

**Comment by ghost** [comment]
image
This button is kinda hard to read

File: styles/theme.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ const lightTheme = responsiveFontSizes(
         tokenPairTableHeader: '#C8E6FE',
         connectTopbar: '#E5F3FF',
         activeTab: '#C8E6FE',
-        blueButton: '#C8E6FE',
+        blueContainer: '#C8E6FE',
```
Comment (saito-sv) [inline styles/theme.js:109]: is this style intended to be use for a container? otherwise the name should reflect its use.

**Comment by ghost** [comment]
How is the state of this?

**Pull request merged**